driver-http/src/main/resources/activities/documents-api/http-docsapi-search-basic.yaml

```diff
@@ -165,6 +167,7 @@ blocks:
       phase:  rampup-read
       filter: or
     statements:
+      # where={"$or":[{"match1":{"$lt":1}},{"match2":{"$eq":"match"}},{"match3":{"$eq":true}}]}
```

Comment:
let's also decrease here to two conditions only to align with `and` and  `or-single-match`

Reply:
and same for main of course,